stockflux-securities-master/src/components/securities-table/SecuritiesTable.js

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useCallback, useReducer } from "react";
+import Components from "stockflux-components";
+import { Link } from "react-router-dom";
+import "./SecuritiesTable.css";
+import * as service from "../../services/SecuritiesService";
+import ValidationError from "../../services/ValidationError";
+import Alert, { AlertType } from "../alert/Alert";
+import { TableState } from "../../enums";
```

Comment:
I think can combine TableState and InputFormState into one, because they are almost identical.

Reply:
Yep done now